### PR TITLE
Updated documentation with v5.3.0

### PR DESCRIPTION
**Pull request by Bonymol-aot**

### CODE REVIEW

File: assets/images/templates/choose_fromList.png
Comment (sumesh-aot) [inline assets/images/templates/choose_fromList.png:1]: Can we provide a better screenshot with meaningful template category and templates? The attached one doesn't convey the real purpose

File: assets/images/templates/choose_fromTemplate.png
Comment (sumesh-aot) [inline assets/images/templates/choose_fromTemplate.png:1]: Same here

File: assets/images/templates/template_success.png
Comment (sumesh-aot) [inline assets/images/templates/template_success.png:1]: Here also create a meaningful form, than a Hello World

File: templates.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<article class="docs-article" >
+    <section class="docs-section" id="templates">
+        <div>
+            <h1>Template feature</h1>
```
Comment (sumesh-aot) [inline templates.html]:
```suggestion
            <h1>Templates</h1>
```

**Review by sumesh-aot — COMMENTED** [review]
Please check the comments

File: assets/images/templates/template_success.png
Comment (Bonymol-aot) [reply assets/images/templates/template_success.png:1]: Done

File: assets/images/templates/choose_fromTemplate.png
Comment (Bonymol-aot) [reply assets/images/templates/choose_fromTemplate.png:1]: Done

File: assets/images/templates/choose_fromList.png
Comment (Bonymol-aot) [reply assets/images/templates/choose_fromList.png:1]: Done

**Review by shuhaib-aot — APPROVED** [review]

**Pull request merged**